ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Node/Serialisation.hs

```diff
@@ -75,7 +75,7 @@ instance Serialise ext => SerialiseNodeToNodeConstraints (MockBlock ext) where
       7 {- CBOR-in-CBOR -} + 1 {- encodeListLen 2 -} + hdrSize + bodySize
     where
       hdrSize  = fromIntegral (Lazy.length (serialise hdr))
-      bodySize = simpleBodySize (simpleHeaderStd hdr)
+      bodySize = fromIntegral (simpleBodySize (simpleHeaderStd hdr))
```

Comment:
Why do we need `fromIntegral` here?

Reply:
`estimateBlockSize` supposed to create `SizeInBytes` but `simpleBodySize` gives `Word32`.